Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Calibration/EditorIntrinsicsCalibration.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    public class EditorIntrinsicsCalibration : MonoBehaviour
+    {
+        public RawImage feedImage;
+        public RawImage lastCornersImage;
+        public RawImage heatmapImage;
+        public RawImage cornersImage;
+
+        private int nextChessboardImageId = 0;
+        private CalibrationAPI calibration = null;
+        int boardWidth = 16;
+        int boardHeight = 11;
+        float chessSquareSize = 0.0236f;
+        Texture2D chessboardCorners = null;
+        Texture2D chessboardHeatmap = null;
+        private int cornerScale = 3;
+        private int heatmapWidth = 12;
+        private List<CalculatedCameraIntrinsics> intrinsics;
+
+#if UNITY_EDITOR
+        private void Start()
+        {
+            CalibrationDataHelper.Initialize(out nextChessboardImageId, out var nextArUcoImageId);
+            calibration = new CalibrationAPI();
+
+            for (int i = 0; i < nextChessboardImageId; i++)
+            {
+                var texture = CalibrationDataHelper.LoadChessboardImage(i);
+                if(texture == null ||
```

Comment:
> if(texture == null || [](start = 15, length = 22)

Same general comment about if( -> if (

And also down below having:

            if (chessboardHeatmap)
                CalibrationDataHelper.SaveImage(chessboardHeatmap, "ChessboardHeatmap");

            if (chessboardCorners)
                CalibrationDataHelper.SaveImage(chessboardCorners, "ChessboardCorners");

Become:


if (chessboardHeatmap)
{
                CalibrationDataHelper.SaveImage(chessboardHeatmap, "ChessboardHeatmap");
}

if (chessboardCorners)
{
                CalibrationDataHelper.SaveImage(chessboardCorners, "ChessboardCorners");
}

I think this is the last I'll leave of this but I will hope that this can be fixed up in the rest of the codebase